ROADMAP.md

```diff
@@ -1,56 +1,47 @@
 
-Objective: "Increase coverage of ML frameworks to support previously unsupported customer workloads."
-* Frameworks
-    * Deploy a ONNX model
-    * Explore supporting other model serialization mechanisms for certain frameworks (e.g. saving PyTorch models with dill)
+Objective: "Enable users to deploy latency sensitive models with KFServing."
+* High Performance Dataplane
+    * Enable support for GRPC or similar.
+    * Continue to support existing HTTP Dataplane.
+
+Objective: "Reduce Total Cost of Ownership when deploying multiple underutilized models."
+* GPU Sharing 
+    * Reduce TCO by enabling models of the same framework and version to be co-hosted in a single model server.
+
+### v0.3 Stability (ETA: Dec 15, 2019)
+Objective: "Improve practices around dependency management." 
+* Migrate to Kubebuilder 2.0.
+    * Use Go Modules.
+    * Stop Vendoring dependencies.
+    * Avoid the extremely heavy dependency on Tensorflow.
+* Migrate to Kubernetes 1.15.
+    * Enable LabelSelectors for the Pod Mutation Webhook.
 
-### v0.3 Performance and Stability (ETA: September 1, 2019)
-Objective: "Prevent feature regressions with 80% end-to-end test coverage against a live Cluster."
+Objective: "Prevent feature regressions with greater end-to-end test coverage against a live cluster."
 * Automated End-to-End Tests
     * Execute against a Kubeflow maintained GKE Cluster.
     * Execute against a Kubeflow maintained AKS Cluster.
     * Achieve >80% Test Coverage of Supported Features.
 
-Objective: "Prevent performance regressions across a known set of representative models."
-* Automated Performance Tests 
-    * Define a set of Models to test covering a wide array of usecases and frameworks.
-    * Publish performance results in a temporally comparable way.
-
-Objective: "Improve the Serverless Experience by reducing cold starts/stops to 10 seconds on warmed models."
-* Model Caching
-    * Reduce model download time by caching models from cloud storage on Persistent Volumes.
-* Image Caching
-    * Reduce container download time by ensuring images are cached in all cloud environments.
-* Server Shutdown
-    * Ensure that all model servers shutdown within 10 seconds of not receiving traffic.
-
-Objective: "Simplify user experience with handling credentials for storage backends."
-* Secure storage mechanisms
-    * Implement a simplified user experience with storage backends protected by credentials (e.g. S3/GCS accounts with credentials)
+Objective: "Improve build and release processes to improve the developer experience and avoid regressions."
+* Improve build reliability
+    * Implement build retries.
+    * Reduce PyTorch build time.
+* Automated Image Injection for Model Servers.
+    * Implement new developer commands to deploy kfserving with local images.
+* Improve versioning of XGBoost, SKLearn, and PyTorch
+    * Replace KFServing version with the corresponding framework version.
```

Comment:
Can we add support for Pytorch GPU inference here?

Reply:
Sure. Do you feel this should be a roadmap level item? Just because it's not on the roadmap, doesn't mean we can't slip it in.